common/src/main/java/me/lucko/luckperms/common/http/AbstractHttpClient.java

```diff
@@ -51,4 +52,15 @@ protected Response makeHttpRequest(Request request) throws IOException, Unsucces
         }
         return response;
     }
+
+    public void shutdown() {
```

Comment:
This change was implemented a little differently on the master branch recently -- 5647b2e5c6b13f685f492e49e4af83d7ffa402fa

If you rebase I think this can be removed :)